MediaBrowser.MediaEncoding/Encoder/EncoderValidator.cs

```diff
@@ -101,32 +105,30 @@ private List<string> GetDecoders(string encoderAppPath)
                 "hevc"
             };
 
-            foreach (var codec in required)
-            {
-                var srch = " " + codec + "  ";
+            var found = required.Where(x => output.IndexOf(x, StringComparison.OrdinalIgnoreCase) != -1);
```

Comment:
This somewhat changes original search - it prepended and appended some spaces. Do you have any idea why it did so? Maybe for codecs which might have starting or ending conflicts (e.g. codec1 and codec12). Might be worth keeping this logic...

Reply:
Your are right, there are ex: `flashsv` and `flashsv2`. Fixed this with a regex rather that a hacky whitespace addition.